control/mpc_lateral_controller/README.md

```diff
@@ -202,6 +202,28 @@ Defined in the `steering_offset` namespace. This logic is designed as simple as
 | cf            | double | front cornering power [N/rad]               | 155494.663    |
 | cr            | double | rear cornering power [N/rad]                | 155494.663    |
 
+### Visualizations of the control results (debug information)
+
+Some signals are published as topics to evaluate the quality of the control results. A control researcher can visualize these information using the plotting tool such as PlotJuggler. Note that the 'deviation' and the 'state error' are defined as the difference between the state and the reference. That is state - reference. This definition is different from the conventional sign of the 'state error' defined in the control theory.
+
+`/control/trajectory_follower/lane_departure_checker_node/debug/deviation/longitudinal`
+The longitudinal deviation of the position at the current time.
+
+`/control/trajectory_follower/lane_departure_checker_node/debug/deviation/longitudinal_velocity`
+The longitudinal deviation of the velocity at the current time.
+
+`/control/trajectory_follower/lateral/diagnostic`
+The signals evaluating the quality of MPC.
+Some commonly used signal are listed below:
+| The signal | Number |
+| :------------------------------------------------------------- | :----- |
+| final steering command (MPC + Clamp + LPF) | [0] |
+| MPC result | [1] |
+| MPC after Clamp (MPC + Clamp) | [21] |
+| feedforward term (reference steering angle) | [2] |
+| lateral error w.r.t the modified reference, feeding MPC module | [5] |
+| lateral error w.r.t the original reference, feeding mpc.cpp | [22] |
+
```

Comment:
You probably have to fix this part
If you view it in the generated documents, it is like this.
https://autowarefoundation.github.io/autoware.universe/pr-6268/control/mpc_lateral_controller/#:~:text=/control/trajectory_follower/lateral,mpc.cpp%20%7C%20%5B22%5D%20%7C

Reply:
fixed in 6ba472aa4ea3ab000e2494328e399dc25a5eebda